### DISCUSSION TITLE
Welcome to peerjs-python Discussions!

### DISCUSSION THREAD

**Discussion by ivelin** [ambianic/peerjs-python#42] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gowri-raj** [comment]
Hi,
May I know how to compile this code? As I am new to this whole thing.
Thank you in advance!
